packages/core/src/shader/Shader.ts

```diff
@@ -17,9 +18,21 @@ export class Shader {
     "GL_OES_standard_derivatives",
     "GL_EXT_draw_buffers"
   ];
+  /** @internal */
+  static _shaderLab?: IShaderLab;
 
   private static _shaderMap: Record<string, Shader> = Object.create(null);
 
+  static set shaderLab(parser: IShaderLab) {
+    this._shaderLab = parser;
+  }
+
+  /**
+   * Create a shader
```

Comment:
- `Create a shader`->`Create a shader by shader source.`

- Add @remarks to tell user must enable shaderlab xxxxx.........

Reply:
fixed